docs/content/en/docs/reference/artifacts.md

```diff
@@ -120,15 +120,15 @@ When you run `image-builder` it will pull in all components needed to create ima
 With this tool, when you build an image you get to choose:
 
 * Operating system type (for example, ubuntu)
-* Provider (vsphere, cloudstack, baremetal, ami, nutanix)
+* Provider (vsphere, cloudstack, baremetal, ami, nutanix, snow)
 * Release channel for EKS Distro (generally aligning with Kubernetes releases)
 * vSphere only: configuration file providing information needed to access your vSphere setup
 * CloudStack only: configuration file providing information needed to access your Cloudstack setup
 * AMI only: configuration file providing information needed to customize your AMI build parameters
 * Nutanix only: configuration file providing information needed to access Prism Central
 
 Because `image-builder` creates images in the same way that the EKS Anywhere project does for their own testing, images built with that tool are supported.
-The following procedure describes how to use `image-builder` to build images for EKS Anywhere on a vSphere, Bare Metal, or Nutanix provider.
+The following procedure describes how to use `image-builder` to build images for EKS Anywhere on a vSphere, Bare Metal, Nutanix, or Snow provider.
```

Comment:
```suggestion
The following procedure describes how to use `image-builder` to build images for EKS Anywhere on a vSphere, Bare Metal, or Nutanix provider. For Snow devices, see <link to Snow image build docs>.
```
We currently have separate sections for building vSphere, Bare Metal and Nutanix images. Since the Snow docs are covering that topic, I think we should add a link to those docs here. Sadly, there seems to be something a bit different for each provider, so we can't just do generic instructions.